ui/utils/Enum.js

```diff
@@ -35,3 +35,5 @@ export const EXTENSIONS = {
 }
 
 export const MesheryPatternsCatalog = "meshery-patterns-catalog"
+
+export const INITIAL_GRID_SIZE = { lg : 6, md : 12, xs : 12 };
```

Comment:
Does `INITIAL_GRID_SIZE` variable have any scope of re-usability somewhere else? If there is please, change the name to what type of grid-settings this variable can be assigned to.

Reply:
Yes this is getting used is `extension.js` and `adapter.js`. What do you think about `EXTENSION_GRID_SIZE`.

Reply:
since they are styles/css related, please write it here: https://github.com/meshery/meshery/tree/master/ui/css

The name might be explanatory, e.g.: `LARGE_6_MED_12_GRID_STYLE`, following the breakdown structure, that conveys:
lg or less: 6, 
medium or less: 12